addons/im_livechat/models/res_users_settings.py

```diff
@@ -6,11 +6,13 @@
 class ResUsersSettings(models.Model):
     _inherit = 'res.users.settings'
 
-    livechat_username = fields.Char("Livechat Username", help="This username will be used as your name in the livechat channels.")
+    livechat_username = fields.Char("Livechat Username", groups="im_livechat.im_livechat_group_user", help="This username will be used as your name in the livechat channels.")
```

Comment:
User settings are already limited to the current user only, so the fields on this model probably don't need the groups as it is already not accessible to other users.